spec/factories/state_file_md_intakes.rb

```diff
@@ -129,9 +129,10 @@
     primary_first_name { "Mary" }
     primary_middle_initial { "A" }
     primary_last_name { "Lando" }
-    primary_birth_date { Date.new(1950, 01, 01) } # matches the bday in md_minimal.json
```

Comment:
this is a weird change?

Reply:
Auto-rubocop correct due to rule https://docs.rubocop.org/rubocop/cops_style.html#stylenumericliteralprefix. I have fix rubocop violations on save enabled. Strange that it assumes 01 is octal.

Changing to `1950, 1, 1` takes away the violation.